.github/workflows/lint.yml

```diff
@@ -2,166 +2,43 @@ on:
   workflow_call:
     inputs:
       type:
-        description: "the expected lint entrypoint (derived from language by default)"
-        default: ""
+        description: "Use make or just to install project dependencies"
         required: false
         type: string
-      language:
-        description: "the language to configure"
-        required: true
-        type: string
-      directory:
-        description: "the directory to run the lint in"
-        required: false
-        type: string
-        default: "."
-      continue-on-error:
-        description: "don't mark the job as failed if a lint fails"
-        default: false
-        required: false
-        type: boolean
-      python-version:
-        description: "the version of Python to configure, if any (can be SemVer-formatted)"
-        default: ""
-        required: false
-        type: string
-      go-version:
-        description: "the Go version"
-        default: ""
-        required: false
-        type: string
-      golangci-lint-version:
-        description: "the golangci-lint version for Go linting"
-        default: "v1.50.1"
+      trunk-arguments:
+        description: Extra arguments to pass to trunk
         required: false
         type: string
-      cargo-sort:
-        description: "run cargo-sort as part of Rust linting"
-        default: false
-        required: false
-        type: boolean
-      org-repo-ref:
-        description: "the ref: branch or commit to use of the org repo"
-        default: "main"
+      trunk-check-mode:
+        description: "Trunk check mode. Leave unset to autodetect changes. Set to 'all' to check the entire repository."
         required: false
         type: string
 
-env:
-  # Rust: GitHub Actions supports color codes, so always enable them.
-  CARGO_TERM_COLOR: always
-
-  # Python: Tell ruff that we're in GitHub Actions, so that it emits annotations.
-  RUFF_FORMAT: github
-
-  # Python: Make-based linting workflows use `INSTALL_EXTRA` to optimize
-  # the subset of development dependencies installed.
-  INSTALL_EXTRA: lint
```

Comment:
@oldsj Let's keep this one (`INSTALL_EXTRA`), since it'll help us optimize Python setup.

Reply:
Actually never mind, I underthought this -- we won't need this because we won't need a `lint` extra anymore. Ignore me!